src/components/utils/utils.js

```diff
@@ -167,6 +167,16 @@ export const getTapChangerEquipmentSectionTypeValue = (tapChanger) => {
     }
 };
 
+export const getTapChangerRegulationTerminalGeneratorValue = (tapChanger) => {
```

Comment:
I'm not sure how tap changers are related to generators ? could you explain ?

Reply:
that was a mistake, sorry